generators/client/templates/angular/src/main/webapp/app/vendor.ts.ejs

```diff
@@ -27,4 +27,45 @@ import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/catch';
 import 'rxjs/add/observable/of';
 import 'rxjs/add/observable/throw';
+
+// Imports all fontawesome core and solid icons
```

Comment:
Doesn't it support web fonts anymore?
Does the SVG + JS approach have a significant saving of bundle size?

Reply:
This is the recommended approach by the FA project. They still support web fonts as well. I guess this is a core project decision, which direction you want to take (if different than the recommended one). I don't have a comparison of the bundle size between the options, but I can figure out if you guys want.

Reply:
The "SVG font" approach seems to have a lot of advantages and it's been a hot topic of discussion, the consensus being that they are better than "Icon Fonts". Here are 3 links about it :

https://www.ianfeather.co.uk/ten-reasons-we-switched-from-an-icon-font-to-svg/
https://cloudfour.com/thinks/seriously-dont-use-icon-fonts/
https://blog.github.com/2016-02-22-delivering-octicons-with-svg/

Especially now that it's supported by FA and easy to setup in our webpack build, I think it's safe to move to this approach.

Reply:
yup, I'm aware of it, I myself use SVG fonts in my projects, however, I was concerned about the JS part of if you look at some other SVG font libs they don't use JS. But then its not a huge concern

Reply:
so lets do it

Reply:
Also seems they will release a Angular version soon https://fontawesome.com/get-started/advanced-options

Reply:
we could migrate to the angular version when released